src/main/java/io/swagger/swaggerhub/plugin/requests/UploadRequest.java

```diff
@@ -36,7 +36,7 @@ public boolean isPrivate() {
 
     public String getOas() { return oas; }
 
-    private SwaggerHubRequest(Builder builder) {
+    private UploadRequest(Builder builder) {
```

Comment:
ah! I see you have it private here, so you must have a reason for making it public above